src/Microsoft.IdentityModel.Tokens/InMemoryCryptoProviderCache.cs

```diff
@@ -234,7 +242,7 @@ public override bool TryRemove(SignatureProvider signatureProvider)
 
         /// <summary>
         /// Calls <see cref="Dispose(bool)"/> and <see cref="GC.SuppressFinalize"/>
-        /// Note: the EventBasedLRUCache is no longer being disposed of, but since this is a public class and can be used as base class of
+        /// Note: the cache is no longer being disposed of, but since this is a public class and can be used as base class of
```

Comment:
If we no longer have IDisposable on the LRU/Hash cache, do we still need to have IDisposable on the InMemoryCryptoProviderCache?

Reply:
Yeah, you are right, IDisposable is not needed any more.